OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -98,10 +98,9 @@ public void ReplacePalette(string name, IPalette pal)
 
 		List<IFinalizedRenderable> GenerateRenderables()
 		{
-			var actors = World.ScreenMap.ActorsInBox(Viewport.TopLeft, Viewport.BottomRight)
+			var actors =
+				World.ScreenMap.ActorsInBox(Viewport.TopLeft, Viewport.BottomRight)
```

Comment:
Seems a bit redundant, considering that there are several even longer lines in this method.